components/browser/engine-gecko/src/main/java/mozilla/components/browser/engine/gecko/ext/CreditCard.kt

```diff
@@ -20,7 +17,7 @@ fun Autocomplete.CreditCard.toCreditCard() = CreditCard(
     number = number,
     expiryMonth = expirationMonth,
     expiryYear = expirationYear,
-    cardType = CARD_TYPE_PLACEHOLDER
+    cardType = number.creditCardIIN()?.creditCardIssuerNetwork?.name ?: ""
```

Comment:
Shouldn't the cartType be a `CreditCardIssuerNetwork` object?

Reply:
Ya that could work here.

Reply:
I didn't carry out this change because I wasn't sure if we want `concept-engine` to have a dependency on `support-utils`. Will seek further guidance.

Reply:
> I didn't carry out this change because I wasn't sure if we want `concept-engine` to have a dependency on `support-utils`. Will seek further guidance.

We definitely don't want to do that. We could make this `CreditCardIssuerNetwork` live in the engine or if it's a concept that is shared between multiple dependencies (and it seems like it is with the sync storages), create a new `concept` component that can hold them. This would flatten the dependency tree as well.

I didn't notice this earlier, but it is a bit odd for credit card data classes to be in the `suppport-utils` component.